kernelci/dashboard
# feat(HardwareDetails): add platform card to tabs

Closes #678

## How to test
- Add platform filter for one tab and verify that the values match
- With a filter selected for one tab, try changing tabs
- Add filter for multiple tabs and verify that all values correctly match
- With a filter selected, try going for a test or build details and verify that coming back to hardware details keep the filters
- Add other filters and verify that they work correctly
- Try doing the same steps in tree details and verify that the behaviour is still the same (this is recommended since some filter methods that are used in tree details were changed)

[Example of Hardware Details with more than one platform (localhost)](http://localhost:5173/hardware/amlogic%2Ca311d?currentPageTab=global.boots&endTimestampInSeconds=1734962400&startTimestampInSeconds=1734530400)

## Visual Reference
![Builds tab](https://github.com/user-attachments/assets/e7a271bd-9922-4819-a743-8e850680731f)

![Boots tab](https://github.com/user-attachments/assets/9841c0d3-7115-4902-ba44-17024fa52e36)

![Tests tab](https://github.com/user-attachments/assets/aafc7e57-309a-4836-916b-683a92fd863d)
